### DISCUSSION TITLE
Q&A: DotLottie for solidjs?

### DISCUSSION THREAD

**Discussion by stabldev** [LottieFiles/dotlottie-web#223] [General]
Hi, do you guys plan on supporting dotlottie component for `solid` js library?

**Comment by theashraf** [comment]
We don't have a plan yet, as we are currently focusing on the stability and full feature support for all dotLottie players

**Comment by theashraf** [comment]
@moonlitgrace, let me know if you're interested in contributing to dotlottie-web by adding the `dotlottie-solid` package. I'm happy to guide and help.

**Reply by stabldev** [reply]
Hi @theashraf, yes it will be much better to be in this monorepo than as a standalone package.
I will send you a PR soon. cheers!

**Comment by stabldev** [comment]
@theashraf hello, is there any ETA on releasing solid support? so that I can archive and deprecate that (dotlottie-solid) package.
thanks!

**Reply by theashraf** [reply]
@moonlitgrace We're planning to make a new release end of this week

**Comment by theashraf** [comment]
@moonlitgrace The dotlottie-solid integration has been released on NPM! Thanks for your contribution 🚀
https://www.npmjs.com/package/@lottiefiles/dotlottie-solid
